### ISSUE TITLE
Impossible dependencies on Ubuntu 15.04 nightly .deb

### ISSUE THREAD

**Issue by guineawheek** [mgba-emu/mgba#19]
When running ```dpkg -i``` on Ubuntu 15.04 using the nightly .debs, dpkg complains of unmet dependencies:
```
Selecting previously unselected package libmgba.
(Reading database ... 180943 files and directories currently installed.)
Preparing to unpack libmgba.deb ...
Unpacking libmgba (0.3.0) ...
dpkg: dependency problems prevent configuration of libmgba:
 libmgba depends on libavcodec54 | libavcodec-extra-54; however:
  Package libavcodec54 is not installed.
  Package libavcodec-extra-54 is not installed.
 libmgba depends on libavformat54; however:
  Package libavformat54 is not installed.
 libmgba depends on libavresample1; however:
  Package libavresample1 is not installed.
 libmgba depends on libavutil52; however:
  Package libavutil52 is not installed.
 libmgba depends on libswscale2; however:
  Package libswscale2 is not installed.
 libmgba depends on libmagickwand5; however:
  Package libmagickwand5 is not installed.

dpkg: error processing package libmgba (--install):
 dependency problems - leaving unconfigured
Errors were encountered while processing:
 libmgba
```
Looking further into the issue, it seems that there are no packages in the Ubuntu vivid repositories that satisfy those exact requirements, as there are very similarly-named packages present, but none of them fulfill the ```dpkg``` requirements

**Comment by endrift** [comment]
Hmm, very strange. The 2015-05-10 one is packaged correctly(ish) and should install. Did the 2015-05-11 one get built against the wrong sysroot...?

**Comment by endrift** [comment]
Fixed now, and I forced a nightly build just to make sure a working version is up.

**Comment by Mowalle** [comment]
Sorry to hijack an old issue, but the same error appears on Ubuntu 15.10. When I run `dpkg -i`, I get

```
Selecting previously unselected package libmgba.
(Reading database ... 297705 files and directories currently installed.)
Preparing to unpack libmgba.deb ...
Unpacking libmgba (0.5-3534-6c1daa9) ...
dpkg: dependency problems prevent configuration of libmgba:
 libmgba depends on libavcodec56 | libavcodec-extra-56; however:
  Package libavcodec56 is not installed.
  Package libavcodec-extra-56 is not installed.
 libmgba depends on libavformat56; however:
  Package libavformat56 is not installed.
 libmgba depends on libavresample2; however:
  Package libavresample2 is not installed.
 libmgba depends on libavutil54; however:
  Package libavutil54 is not installed.
 libmgba depends on libswscale3; however:
  Package libswscale3 is not installed.
 libmgba depends on libzip2; however:
  Package libzip2 is not installed.

dpkg: error processing package libmgba (--install):
 dependency problems - leaving unconfigured
Processing triggers for hicolor-icon-theme (0.15-0ubuntu1) ...
Errors were encountered while processing:
 libmgba
```

The problem is that `libavcodec56` etc. are not even included in the Ubuntu 15.10 repositories. It only offers libavcodec-ffmpeg56 instead.
